README.md

```diff
@@ -1,5 +1,3 @@
-# Note - as of Nov 7, 2022, the main branch of this repository is frozen. The develop branch is used for all releases and all developments.
-
```

Comment:
I think this can stay in develop.

Reply:
So far we've only added it to the main/master branches, not the develop branches ...

Reply:
Makes sense, thank you!